package.json

```diff
@@ -20,12 +20,12 @@
     "d3-axis": "3.0.0",
     "d3-scale": "3.3.0",
     "d3-selection": "3.0.0",
-    "eslint": "8.34.0",
+    "eslint": "8.38.0",
     "eslint-plugin-compat": "4.1.1",
     "eslint-plugin-playwright": "0.12.0",
     "eslint-plugin-vue": "9.9.0",
     "eslint-plugin-you-dont-need-lodash-underscore": "6.12.0",
-    "eventemitter3": "1.2.0",
+    "eventemitter3": "4.0.7",
```

Comment:
This is a pretty large jump with multiple breaking changes (as both of our unit and e2e test runs are failing). Could you look into what changes need to be made in order to facilitate the upgrade to v4.x.x?

Otherwise, we could exclude the `eventemitter3` upgrade from this PR and do this work in a follow-up PR (we already have a separate issue open to upgrade this dependency [here](https://github.com/nasa/openmct/issues/4656)).

Reply:
Hi @ozyx! Sure. Since we already have another issue, I'll exclude it from this PR, and review what changes exist between these versions to make easier the change in #4656

Reply:
Awesome. Thanks so much!